src/baserow/client.py

```diff
@@ -170,12 +191,6 @@ def list_database_table_rows(
       page + 1 if response['next'] else None,
       response['results'])
 
-  def create_database_table_row(self, table_id: int, record: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
-    return self._request('POST', f'/api/database/rows/table/{table_id}/', json=record).json()
-
-  def update_database_table_row(self, table_id: int, row_id: int, record: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
-    return self._request('PATCH', f'/api/database/rows/table/{table_id}/{row_id}/', json=record).json()
-
```

Comment:
I think you accidentally dropped this addition during the merge 😮

Reply:
It's because I pulled your latest commits from develop, so I can add the entry to the `.changelog/_unreleased.yml` file, which did not exist when I first forked the repo

Reply:
No, you are right, that's on me